### ISSUE TITLE
Indentation is missing during conversion

### ISSUE THREAD

**Issue by erenatas** [ktdreyer/jenkins-job-wrecker#112]
Hello,

We have realized that the indentation of multiline values have been missing due to 

https://github.com/ktdreyer/jenkins-job-wrecker/blob/26789991e5cfc6af89d02fd99b9a7feb9f694d81/jenkins_job_wrecker/cli.py#L29-L38

As it strips the lines, therefore deletes the whitespaces in the beginning and end, which is an unintended situation especially on scripts. For example:

``` xml
<shell>if [[ test.txt ]]; then
    echo "hello"
fi</shell>
```

Should be converted as:

``` yaml
shell: |
    if [[ test.txt ]]; then
        echo "hello"
    fi
```

But it's converted as:
``` yaml
shell: |
    if [[ test.txt ]]; then
    echo "hello"
    fi
```

I have realized that the problem is actually not caused by trailing whitespaces, it's caused by `\r` **(carriage return)** which is `&#xd;` in XML correspondence. If `&#xd;` is removed from the XML, it works fine with
```
def str_presenter(dumper, data):
  if len(data.splitlines()) > 1:  # check for multiline string
    return dumper.represent_scalar('tag:yaml.org,2002:str', data, style='|')
  return dumper.represent_scalar('tag:yaml.org,2002:str', data)
```

The only part that I have seen a carriage-return appearing so far is in description. Instead of stripping all the lines which **would break indentation for any kind of multiline value**, I believe that we should do this only on description or remove all the `&#xd;` values before starting to parse the XML file. AFAIK, both ways do the work.

There may be a better way for solving this problem and that is why I am opening this ticket. **If you have any ideas, please leave a comment.**

**Comment by ktdreyer** [comment]
The comment in the code says that it is trying to avoid *trailing* whitespace, but we're calling plain `strip()` here, which removes both *leading* and *trailing* whitespaces. What happens when you change that to `rstrip()`?

**Comment by erenatas** [comment]
> The comment in the code says that it is trying to avoid _trailing_ whitespace, but we're calling plain `strip()` here, which removes both _leading_ and _trailing_ whitespaces. What happens when you change that to `rstrip()`?

Sorry for the late reply. While I was investigating, I thought that rstrip did not do the job, but now I testing it again, It seems it works just fine! I will open a PR. Thank you so much for the suggestion!

Cheers!

**Comment by ktdreyer** [comment]
Cool, thanks! Are you ok with closing this PR, in that case?

**Comment by erenatas** [comment]
I will open a new PR, added a test.
